backend/substrapp/compute_tasks/outputs.py

```diff
@@ -57,11 +57,13 @@ def _save_performance(self, output: context.OutputResource):
     def _save_models(self, models: list[context.OutputResource]):
         logger.info("saving models")
 
+        # Save models to minio
         new_models = []
         for model in models:
             new_model = self._save_model_to_local_storage(model)
             new_models.append(new_model)
 
+        # Register models in the orchestrator
         try:
             with get_orchestrator_client(self._ctx.channel_name) as client:
                 registered_models = client.register_models({"models": new_models})
```

Comment:
Saving to the orchestrator could be delegated to the API as for other assets, or is there anything preventing this ?

Reply:
We register performance above directly by calling the orchestrator so I kept it there to be consistent